libraries/chain/include/graphene/chain/protocol/chain_parameters.hpp

```diff
@@ -46,6 +46,7 @@ namespace graphene { namespace chain {
       optional < uint32_t >           son_vesting_amount;
       optional < uint32_t >           son_vesting_period;
       optional < uint32_t >           son_pay_daily_max;
+      optional < account_id_type >    son_btc_account;
```

Comment:
Remove son_btc_account.

Reply:
This is required for consensus among SONs. Please refer to the LLD.